### PR TITLE
Stripe: Update Invoice properties to match version 2019-03-14

### PR DESCRIPTION
**Pull request by reacuna**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://stripe.com/docs/api/invoices/object
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/stripe/index.d.ts [hunk 1]
```diff
@@ -1956,9 +1956,17 @@ declare namespace Stripe {
...
+             * The fee in cents that will be applied to the invoice and transferred to the application owner's
+             * Stripe account when the invoice is paid.
+             */
+            application_fee: number;
+
             /**
              * The fee in pence that will be applied to the invoice and transferred to the application owner’s
              * Stripe account when the invoice is paid.
+             *
+             * Changed from `application_fee` on Stripe API Version 2019-03-14
```
Comment (LinusU) [inline types/stripe/index.d.ts]: maybe add `@deprecated` here? 🤔

Comment (reacuna) [reply types/stripe/index.d.ts]: You mean on `application_fee`, right?

**Comment by PranavSenthilnathan** [comment]
@reacuna So as far as I understand the added properties are not due to the client package getting updated but rather the Stripe server apis changing? Basically I'm wondering whether the version in the index.d.ts should be updated.

**Comment by reacuna** [comment]
@PranavSenthilnathan yes, you are right. I didn't change the version in index.d.ts because of that. However, this PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35002 does change the version

**Pull request merged**